certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2291,8 +2305,9 @@ public class Script : ScriptBase
       var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
       JObject newBody = new JObject();
 
-      JArray DocumentRecord = (body["envelopes"] as JArray) ?? new JArray();
+      JArray envelopes = (body["envelopes"] as JArray) ?? new JArray();
       JArray filteredRecords = new JArray();
+      JArray DocumentRecord = new JArray();
```

Comment:
fix variable name casing to `documentRecords`

Reply:
fixed